ui-tests/src/test/java/org/openmrs/reference/page/AdministrationPage.java

```diff
@@ -21,7 +21,8 @@
     private static final By MANAGE_PROVIDERS = By.cssSelector("#content a[href='/openmrs/admin/provider/index.htm']");
     private static final By MANAGE_PERSONS = By.cssSelector("#content a[href='/openmrs/admin/person/index.htm']");
     private static final By MANAGE_HTMLFORMS_PAGE = By.cssSelector("#content a[href*='/module/htmlformentry/htmlForms.list']");
-
+    private static final By REPORT_ADMINISTRATION_PAGE = By.cssSelector("#menu > li:nth-child(1) > a");
```

Comment:
is `REPORT_ADMINISTRATION_PAGE `a link, button or what?

Using child(x)-value usually breaks when UI changes comes in so I would use another element reference that can hardly break.